### ISSUE TITLE
Support Amazon's ASIN product identifier

### ISSUE THREAD

**Issue by BobBreukhoven** [Yoast/wpseo-woocommerce#862]
https://yoast.atlassian.net/browse/DUPP-762

[Schema.org](http://schema.org/) is in the process of adding support for `ASIN` values (Amazon Standard Identification Number) on `Product/Offer` nodes.

> Further context here: [Proposal to add support for Amazon's ASIN product identifierQUEUED FOR STAGING (WEBSCHEMAS.ORG)](https://github.com/schemaorg/schemaorg/issues/3123)

We should therefore:

- Add UI inputs for this in Woo and Shopify, alongside our existing inputs for GTIN (and similar).
- Output an asin attribute and value as an additional property, alongside anywhere where we would output a gtin attribute (on Product and/or Offer nodes).

> Do we have validation mechanisms for these inputs? If so, we should check that the value provided is a 10-character alphanumeric string.

> This should not influence / by assessed by the Product Identifier assessment (which is designed to avoid warnings around missing GTIN/similar values; which providing an asin won’t currently address).

### User story

- As a merchant selling (or affiliating) products which have Amazon ‘ASIN’ values;
- I want to output those values in my schema;
- So that Google (etc) can better understand my product pages .

#### Short-term

- As a Yoast user, you have some of the best and most cutting-edge structured data on the web.
- Now, [schema.org](http://schema.org/) is planning to support Amazon’s asin product descriptor; so we’ve added support to Yoast SEO.
- If you sell or recommend products on Amazon, you can get ahead of the competition by being an early adopter, and helping Google to better understand your product pages!

**Comment by enricobattocchi** [comment]
Let's check whether Shopify already includes this and if we need to be in sync (I don't think we need on the technical side because it's not going to be in parts of the code that we share)

**Comment by enricobattocchi** [comment]
It's also part of https://github.com/Yoast/roadmap/issues/159, maybe we want to address all those things in the same sprint? @marijnyoast

**Comment by BobBreukhoven** [comment]
Yes, @enricobattocchi , all the Woo updates should be tackled in the same "release", not sure if we want it in the same "sprint". https://github.com/Yoast/roadmap/issues/159 

And Shopify will start working on it, after our current project, hopefully in the upcoming weeks

**Comment by marijnyoast** [comment]
Let's park it again then. Still good to have it digged out of our needs refinement bin.
